### ISSUE TITLE
Implement review pattern for Activity List

### ISSUE THREAD

**Issue by hbillings** [18F/cms-hitech-apd#1501]
There are two review sections necessary to replace the current Activity List/Activity Overview sections. They both use the CMS review pattern. #1452 is the Overview issue.

![image](https://user-images.githubusercontent.com/23194720/58493520-1cf22780-8141-11e9-8b0a-0ed2a92a3f69.png)

### This task is done when...
 - [ ] There is one expanded section by default
 - [ ] Expanded activity sections have two data entry points, an activity name text box and a type radio button
- [ ] Expanded activities may be completed or cancelled, with the exception of the first one
- [ ] "Add another activity" is visible
- [ ] When collapsed, the list follows the [CMS review pattern](https://design.cms.gov/patterns/review/)

**Comment by austinhernandez** [comment]
When we reviewed this section together as a team 4/22 we decided we were going to keep the current functionality of the `Activity List` the same as it is currently, so the above mockup is incorrect. We will be creating the list of activities first, then populating the sections below.

Attached is how the activity list should look with the new styles.

![image](https://user-images.githubusercontent.com/23194720/58493520-1cf22780-8141-11e9-8b0a-0ed2a92a3f69.png)
